backend/src/database/migrations/20230623093632-add-userid-column-to-govsg-tx-message.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+module.exports = {
```

Comment:
I think this one can merge with the prior migrations (from my other PR) as we haven't ran the migrations in prod, lemme do that

Reply:
let me know when you're done, then I'll rebase

Reply:
done